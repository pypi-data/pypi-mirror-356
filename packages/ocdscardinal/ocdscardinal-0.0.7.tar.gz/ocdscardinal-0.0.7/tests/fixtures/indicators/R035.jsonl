{"ocid":"F1","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]}]}
{"ocid":"F2","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"disqualified","tenderers":[{"id":"E"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]}]}
{"ocid":"repeat","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"V"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]},{"status":"active","suppliers":[{"id":"V"}]}]}
{"ocid":"accept","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"invited","tenderers":[{"id":"I"}]},{"status":"pending","tenderers":[{"id":"P"}]},{"status":"withdrawn","tenderers":[{"id":"W"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]},{"status":"cancelled","suppliers":[{"id":"C"}]},{"status":"unsuccessful","suppliers":[{"id":"U"}]}]}
{"ocid":"pending award","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]},{"status":"pending"}]}
{"ocid":"invalid award","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]},{"status":"invalid"}]}
{"ocid":"valid_tenderer_ids.len() = 0","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]}]}
{"ocid":"valid_tenderer_ids.len() > 1","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"V"}]},{"status":"valid","tenderers":[{"id":"W"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]}]}
{"ocid":"valid_tenderer_ids ! award_supplier_ids","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"S"}]}]}
{"ocid":"valid_tenderer_ids < award_supplier_ids","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]},{"status":"active","suppliers":[{"id":"S"}]}]}
{"ocid":"difference < threshold","bids":{"details":[{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]}]}
{"ocid":"cancelled tender","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"D"}]},{"status":"valid","tenderers":[{"id":"V"}]}]},"awards":[{"status":"active","suppliers":[{"id":"V"}]}],"tender":{"status":"cancelled"}}
