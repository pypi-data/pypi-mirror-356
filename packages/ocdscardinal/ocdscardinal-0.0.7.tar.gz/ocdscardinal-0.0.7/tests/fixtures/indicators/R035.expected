{"OCID":{"F1":{"R035":1.0},"F2":{"R035":2.0},"repeat":{"R035":1.0},"accept":{"R035":1.0}},"Tenderer":{"V":{"R035":0.0}}}
